sys/targets/targets.go

```diff
@@ -65,14 +67,23 @@ type osCommon struct {
 	CPP string
 	// Common CFLAGS for this OS.
 	cflags []string
+	// Default compiler.
+	defaultCompiler string
 }
 
-func Get(OS, arch string) *Target {
+func Get(OS, arch, compiler string) *Target {
```

Comment:
This change will cause tremendous amount of plumbing. This function is called in 33 places and transitively in just any binary. If we do this approach with binary flag and explicit argument, we will need to change to change all binaries and plumb it though lots of intermediate places. And in the end it's always coming from SYZ_CLANG and passed in exactly the same way (hopefully).
I think we just need to directly check SYZ_CLANG env var in sys/targets. This will be like 1 line instead of a hundred of changed files.

Reply:
Right, that's why Travis complained. I'll check SYZ_CLANG instead.